### ISSUE TITLE
assigning values to symbols in "eval" and "run" by passing assignment object

### ISSUE THREAD

**Issue by davidedc** [davidedc/Algebrite#37]
numerically solve expression by assigning values to symbols: we can now pass additional expressions to eval() or run() to do that:

```Algebrite.run('x^2+y^2', 'x=2', 'y=3')  // => 13 ```

It'll be great if it can be done also like math.js do:

```Algebrite.run('x^2+y^2', { x: 2, y: 3 })```

**Comment by aperep** [comment]
Yes, this is a major obstacle in using Algebrite. Ideally, one needs to be able to compile a function into JS like it is done in math.js.

**Comment by davidedc** [comment]
Indeed Algebrite can compile to JS, but it's through a strange method called "findDependencies" or similar, you could look into that perhaps. But we miss the "assignment object" thing.

So, I mean, it's two separate things: JS code generation is there but assignment from object is not.
